fmriprep/utils/static_files.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
```

Comment:
Ok I removed all the code I copied over in favor of the version in fmriprep/data e2488eb